docs/api/annotations.md

```diff
@@ -0,0 +1,233 @@
+
+
+## Get a single annotation
+
+### URL
+
+```http
+GET /annotation/:annotation HTTP/1.1
+```
+
+* `annotation` - **Required** The annotation identifier.
+
+### Request
+
+```http
+GET /annotation/annotation_1 HTTP/1.1
+Accept: application/json
+```
+
+### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  ...
+}
+```
+
+
+## Create an annotation
+
+### URL
+
+```http
+POST /annotations HTTP/1.1
+```
+
+### Request parameters
+
+Parameters should be posted as JSON body.
+
+Name             | Type     | Description
+-----------------|----------|------------
+`id`             | `string` | **Optional** The unique identier of the annotation. If none is given, one is generated.
+`doneBase`       | `string` | **Optional** The datetime this annotation was marked as done by the base annotator.
+`doneFull`       | `string` | **Optional** The datetime this annotation was marked as done by the full annotator.
+`recording`      | `string` | **Required** The reference to the recording (type
+`FlatRecordingReference` as specified below.
+`assignment`     | `string` | **Required** The unique identifier of the assignment this annotation
+is for.
+`fragments`      | `list`   | **Required** The annotated fragments (type `Fragment` as specified below).
+
+The Recording format (`FlatRecordingReference`).
+
+Parameters should be posted as JSON body.
+
+Name             | Type     | Description
+-----------------|----------|------------
+`tenant`         | `string` | **Required** The unique identier of the tenant.
+`organisation`   | `string` | **Required** The unique identifier of the organisation.
+`recording`      | `string` | **Required** The unique identifier of the recording.
+`challenge`      | `string` | **Required** The unique identifier of the challenge.
+
+
+The Fragment format:
+
+Parameters should be posted as JSON body.
+
+Name     | Type    | Description
+---------|---------|------------
+`start`  | `float` | **Required** The start point of the fragment, in seconds.
+`end`    | `float` | **Required** The end point of the fragment, in seconds.
+`labels` | `list`  | **Required** The labels assigned to this fragment (type `Label` as specified
+below).
+
+
+The Label format:
+
+Parameters should be posted as JSON body.
+
+Name      | Type     | Description
+----------|----------|------------
+`layer`   | `string` | **Required** The specifier for the layer (Words, OutOfPrompt, Error, Noise).
+`label`   | `string` | **Required** The specifier of a particular label within a layer (Toevoeging ZN, Toevoeging Lidwoord, etc).
+`comment` | `string` | **Required** An optional extra value (e.g. when the label is Toevoeging Lidwoord: De, Het, Een).
+
+
+## Update an annotation
+
+### URL
+
+```http
+PUT /annotations/:annotation HTTP/1.1
+```
+
+* `annotation` - **Required** The annotation identifier.
+
+Update one or more properties of an existing annotation.
+
+### Request parameters
+
+XXX
+
+
+### Request
+
+XXX
+
+### Response
+
+XXX
```

Comment:
I think it is good to allow merging of this PR only if also these parts are in (otherwise they will end up on the pile of work todo). Should not be much different as the POST.

Reply:
Will remove XXX. Although I don't feel like copy-pasting the POST request and response if it's really the same body. Make documentation maintenance more work.

Reply:
Agree on the more maintenance part. Only thing here is that we have done it everywhere. It is better to keep the docs consistent at this point (as far as they are..).